index.html

```diff
@@ -8,305 +8,357 @@
+              Frameworks
+              <img src="images/open.png" alt="Right arrow" />
+            </div>
+          </li>
+          <hr />
+          <li class="knowledge-name">
+            <div class="knowledge-item">
+              Skills
+              <img src="images/open.png" alt="Right arrow" />
+            </div>
+          </li>
+          <hr />
+        </ul>
+      </div>
+    </section>
 
-              <button type="button" class="get-resume">Get my resume</button>
-          </div>
-          <div class="knowledge">
-              <ul>
-                  <li class="knowledge-name">
-                      <div class="knowledge-item">
-                          Languages
-                          <img src="images/down-arrow.svg" alt="Down arrow" />
-                      </div>
-                      <ul>
-                          <li class="knowledge-summary">
-                              <img src="images/js.svg" alt="Javascript icon" />
-                              <a href="#JS">Javascript</a>
-                          </li>
-                          <li class="knowledge-summary">
-                              <img src="images/html.svg" alt="HTML icon" />
-                              <a href="#html">HTML</a>
-                          </li>
-                          <li class="knowledge-summary">
-                              <img src="images/css.svg" alt="CSS icon" />
-                              <a href="#CSS">CSS</a>
-                          </li>
-                      </ul>
-                  </li>
-                  <li class="knowledge-name separate-style">
-                      <div class="knowledge-item">
-                          Frameworks
-                          <img src="images/right-arrow.svg" alt="Right arrow" />
-                      </div>
-                  </li>
-                  <hr />
-                  <li class="knowledge-name">
-                      <div class="knowledge-item">
-                          Skills
-                          <img src="images/right-arrow.svg" alt="Right arrow" />
-                      </div>
-                  </li>
-                  <hr />
-              </ul>
-          </div>
-      </section>
+    <section class="contact-form">
+      <div class="contact-tittle">
+        <h2 class="contact-me-text">Contact me</h2>
+      </div>
+      <div class="contact-main-text">
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
 
+      <form action="https://formspree.io/f/xayzjewb" method="post">
+        <input
+          id="username"
+          type="text"
+          name="username"
+          required
+          maxlength="30"
+          title="Name should be text and not more than 30 characters"
+          placeholder="Alexandra Loremipsum"
+        />
```

Comment:
- [ ] You've done a great job styling the works section. However, it would be even better to ensure that the text below the title 'Contact Me' matches the design, which has three lines instead of two :+1:.

**Your design**

![Screenshot from 2023-03-03 08-02-46](https://user-images.githubusercontent.com/105117832/222729064-7b7b0afe-873b-4329-80bd-8d154ce4527f.png)

**Your design 175%**

![Screenshot from 2023-03-03 08-03-17](https://user-images.githubusercontent.com/105117832/222729143-cf255878-c9c1-4fb1-b08c-64f71923fe01.png)

**Figma**

![Screenshot from 2023-03-03 08-02-27](https://user-images.githubusercontent.com/105117832/222729182-d974713d-aced-4ed7-8316-ad7ab4bbd716.png)